### PR TITLE
Update local-setup documentation

### PR DESCRIPTION
**Pull request by tatilepizs**
Update how to bypass denial page mac+chrome

### CODE REVIEW

File: content/en/apps/tutorials/local-setup.md [hunk 1]
```diff
@@ -37,15 +37,15 @@ To read more about these concepts, see our [Docker Setup guide]({{< relref "core
 
 Before you begin, you need to have some useful software and tools that are required for things to work:
 
-* [nodejs](https://nodejs.org/en/) 8 or later
+* [nodejs](https://nodejs.org/en/) from version 8 to 12
```
Comment (jkuester) [inline content/en/apps/tutorials/local-setup.md:40]:
```suggestion
* [nodejs](https://nodejs.org/en/) version 12 or later
```

I believe that now at least Node 12 is required for _building_ cht-core.

**Review by jkuester — CHANGES_REQUESTED** [review]
+1  Thanks for helping to keep these docs up to date!  (Just one minor change suggested)

File: content/en/apps/tutorials/local-setup.md (hunk 1 above)
Comment (tatilepizs) [reply content/en/apps/tutorials/local-setup.md:40]: Does it work with a node version higher than 12? I thought that we couldn't install the latest one

Comment (jkuester) [reply content/en/apps/tutorials/local-setup.md:40]: So, I am pretty sure that it _should_ work with higher versions and I know that folks have tried it before with Node 16 (and I thought they got it working).  But, just now I checked the Node versions that we use in our [CI builds on GitHub actions](https://github.com/medic/cht-core/blob/master/.github/workflows/build.yml) and I see that we only try to build the project with Node 12 and then test running it with Node 8, 10, and 12.  So, you are probably right that we should just suggest version 12 here as the recommended version for development (and not have "or later").

Comment (tatilepizs) [reply content/en/apps/tutorials/local-setup.md]: Done, I changed it to version 12

**Review by jkuester — APPROVED** [review]
Looks great!  You can go ahead and merge this PR.

**Pull request merged**